tests/incremental/hashes/function_interfaces.rs

```diff
@@ -278,7 +278,7 @@ pub fn no_mangle() {}
 #[rustc_clean(cfg = "cfail3")]
 #[rustc_clean(cfg = "cfail5")]
 #[rustc_clean(cfg = "cfail6")]
-#[no_mangle]
+#[unsafe(no_mangle)]
```

Comment:
These changes are needed because the "this should be unsafe" lint (in edition <2021) is now produced by the attribute parsing system, and therefore stored in https://github.com/rust-lang/rust/blob/0864097cd31ee30f5081ba588a5c9820c2c6fc71/compiler/rustc_hir/src/hir.rs#L1537

One version of this test produces the lint, the other one does not. The easy solution is to make both versions not produce the lint, by adding `unsafe`